plugins/aap-backend/config.d.ts

```diff
@@ -1,13 +1,4 @@
-import { TaskScheduleDefinitionConfig } from '@backstage/backend-tasks';
-
-interface AapConfig {
-  baseUrl: string;
-  /** @visibility secret */
-  authorization: string;
-  system?: string;
-  owner?: string;
-  schedule?: TaskScheduleDefinitionConfig;
-}
+import { AapConfig } from './src/types';
```

Comment:
Hold on a minute... I'm not sure this works as intended... I'll need to take a look into the build process.

Reply:
How does this get references when there's no `src` folder in the built NPM package and `config.d.ts` is not transpiled into JS? It seems this reference doesn't exist/never gets imported.

Reply:
Have updated it have inline